bears/python/DodgyBear.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from dependency_management.requirements.PipRequirement import PipRequirement
```

Comment:
import sud be in alphabetical order.

Reply:
made the changes ;)

Reply:
similar changes in `DodgyBearTest.py` too?

Reply:
yup ;)

Reply:
changed the import order in the test file too